portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,28 +14,32 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-import java.util.ArrayList;
-import com.google.gson.Gson;
-
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  ArrayList<String> messages = new ArrayList<>();
+  private Map<String, String> messages = new HashMap<String, String>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    messages.add("This is brought to you by JSON");
-    messages.add("JSON is JavaScript Object Notation");
-    messages.add("It is great for storing data like this");
-
     response.setContentType("application/json;");
     String json = new Gson().toJson(messages);
```

Comment:
No change needed, just FYI:
Google protocol buffers is a much better alternative to XML and JSON.

Reply:
Ok that's useful information for later web development, I will keep this in mind.